src/modules/launcher/Plugins/Microsoft.Plugin.Folder/Main.cs

```diff
@@ -66,27 +60,39 @@ public List<Result> Query(Query query)
                 throw new ArgumentNullException(paramName: nameof(query));
             }
 
-            var results = GetUserFolderResults(query);
+            var results = GetFolderPluginResults(query);
 
-            string search = query.Search.ToLower(CultureInfo.InvariantCulture);
-            if (!IsDriveOrSharedFolder(search))
+            // todo why was this hack here?

```

Comment:
Is this hack still appropriate?

Reply:
I am not sure why this was added. My best would guess would be to put them above indexer plugin results.